MiniApp/en.lproj/Localizable.strings

```diff
@@ -19,3 +19,7 @@
 "wait_message" = "Please wait...";
 "ok" = "OK";
 "cancel" = "Cancel";
+
+"test_mode_publishing" = "Production";
```

Comment:
Should this be "Published" instead of "Production"? I think that the backend is distinguishing it as "Published" vs. "Test" Mini Apps.

Reply:
sure, you're right